docs/metadataPackage.md

```diff
@@ -0,0 +1,99 @@
+# metadataPackage JSON File
+
+The input to our `dr2-ingest` Step Function is a JSON file describing the package to be ingested below the series. The JSON file contains a single array of objects, where each object represents a folder, Asset, or File within the package.
```

Comment:
1. Hmm, maybe "The input to our `dr2-ingest` Step Function is a S3 location to a JSON file..."
2. Maybe instead of "ingested below the series" either "ingested below the series folder/directory" or "ingested into the series folder/directory"?